client/extensions/woocommerce/app/reviews/review-card.js

```diff
@@ -29,43 +33,141 @@ class ReviewCard extends Component {
 			review: PropTypes.string,
 			rating: PropTypes.number,
 		} ).isRequired,
+		currentStatus: PropTypes.string.isRequired,
 	};
 
-	render() {
+	state = {
+		isExpanded: false,
+	};
+
+	toggleExpanded = () => {
+		this.setState( ( { isExpanded } ) => ( { isExpanded: ! isExpanded } ) );
+	}
+
+	renderToggle() {
+		const { isExpanded } = this.state;
+		return (
+			<Button
+				borderless
+				className="reviews__action-collapse"
+				onClick={ isExpanded ? this.toggleExpanded : noop }
+			>
+				<Gridicon icon="chevron-down" />
+			</Button>
+		);
+	}
+
+	renderGravatar() {
 		const { review } = this.props;
-		const classes = classNames( 'reviews__card', {
-			'is-approved': review.approved,
-			'is-unapproved': ! review.approved,
-		} );
 		const author = {
 			avatar_URL: review.avatar_urls[ 48 ],
 			display_name: review.name,
 		};
+		return <Gravatar user={ author } />;
+	}
 
+	renderProductImage() {
+		const { review } = this.props;
 		const productImageClasses = classNames( 'reviews__product', { 'is-placeholder': ! review.product.image } );
+		return (
+			<div className={ productImageClasses }>
+				{ review.product.image && ( <img src={ review.product.image } /> ) }
+			</div>
+		);
+	}
 
+	renderActionsBar() {
+		const { review, currentStatus } = this.props;
 		return (
-			<Card className={ classes }>
-				<div className={ classNames( 'reviews__header', 'is-preview' ) } >
+			<div className={ classNames( 'reviews__header' ) } >
+				<ReviewActionsBar review={ review } currentStatus={ currentStatus } />
+				{ this.renderToggle() }
+			</div>
+		);
+	}
+
+	renderPreview() {
+		const { review } = this.props;
+		return (
+			<div
+				className={ classNames( 'reviews__header', 'is-preview' ) }
+				onClick={ this.toggleExpanded }
+			>
+				<div className="reviews__author-gravatar">
+					{ this.renderGravatar() }
+				</div>
+				<div className="reviews__info">
+					<div className="reviews__author-name">
+						{ review.name }
+						{ review.verified && <Gridicon icon="checkmark-circle" size={ 18 } /> }
+					</div>
+					<div className="reviews__date">{ humanDate( review.date_created_gmt + 'Z' ) }</div>
+				</div>
+				<AutoDirection>
+					<div className="reviews__content">
+						{ decodeEntities( stripHTML( review.review ) ) }
+					</div>
+				</AutoDirection>
+				<div className="reviews__rating">
+					<Rating rating={ ( review.rating / 5 ) * 100 } />
```

Comment:
Nit. Old developer here. If you move the 100 to the left, multiply it by review.rating and then divide by 5 you can ditch the parens. If you feel especially cray cray, just multiply review.rating by 20. :)